pg_search/src/postgres/customscan/pdbscan/mod.rs

```diff
@@ -147,6 +149,32 @@ impl PdbScan {
             state.custom_state_mut().snippet_generators = snippet_generators;
         }
 
+        if need_snippet_positions {
```

Comment:
Nit: it would be nice to be able to dedupe these two blocks as they appear to be about 95% similar.